src/main/drivers/serial_uart.c

```diff
@@ -39,10 +39,11 @@
 static void usartConfigurePinInversion(uartPort_t *uartPort) {
 #if !defined(INVERTER) && !defined(STM32F303xC)
     UNUSED(uartPort);
-#endif
+#else
+    bool inverted = (uartPort->port.options & SERIAL_INVERTED) == SERIAL_INVERTED;
```

Comment:
You should stick to one idiom when testing option bits. Rest of code uses implicit nonzero test.